.github/workflows/tests.yml

```diff
@@ -21,33 +21,39 @@ on:
 
 jobs:
   tests:
-    environment: mila-deploy
     runs-on: [self-hosted, gpu]
-
+    environment: mila-deploy
```

Comment:
I think we can remove this after all, the existing configuration was already blocking what we wanted to block.